Client/Frontend/Browser/ReaderModeBarView.swift

```diff
@@ -8,6 +8,32 @@ import SnapKit
 
 enum ReaderModeBarButtonType {
     case MarkAsRead, MarkAsUnread, Settings, AddToReadingList, RemoveFromReadingList
+
+    private var description: String {
```

Comment:
I don't think there is anything related to localization. Maybe it is best to simply call it `accessibilityLabel`? That way it is immediatly clear what it is for and similar to `UIAccessibility.accessibilityLabel`.